chat-client/src/contracts/serverContracts.ts

```diff
@@ -1,10 +1,17 @@
-import { TabAddParams, TabChangeParams, TabRemoveParams } from '@aws/language-server-runtimes-types'
+import { ChatParams, TabAddParams, TabChangeParams, TabRemoveParams } from '@aws/language-server-runtimes-types'
+import {
+    chatRequestType,
+    readyNotificationType,
+    tabAddNotificationType,
+    tabChangeNotificationType,
+    tabRemoveNotificationType,
+} from '@aws/language-server-runtimes/protocol'
```

Comment:
Can't we keep the hardcoded method names here or move it to `types` package? The separation of `types` and `runtimes` came from this `chat-client` package only needing the chat types definition, importing the full runtimes now makes that decision unnecessary

Reply:
Makes sense, I will go back to hardcoding for now and move these to the types package in the meantime